kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/core/test/TestCaseSeverityLevel.kt

```diff
@@ -0,0 +1,19 @@
+package io.kotest.core.test
+
+import io.kotest.core.internal.KotestEngineSystemProperties
+import io.kotest.mpp.sysprop
+
+enum class TestCaseSeverityLevel(val level: Int) {
+   BLOCKER(4),
+   CRITICAL(3),
+   NORMAL(2),
+   MINOR(1),
+   TRIVIAL(0);
+
+   fun isEnabled(): Boolean {
+      return level >= getConfigSeverityLevel().level
+   }
+
+   fun getConfigSeverityLevel(): TestCaseSeverityLevel =
+      sysprop(KotestEngineSystemProperties.severityPrefix)?.let { valueOf(it)} ?: TRIVIAL
```

Comment:
It might be cleaner to write:

valueOf(sysprop(KotestEngineSystemProperties.severityPrefix ?: 0)

Reply:
I wouldn't put this fallback inside the enum. I would just have the test case config return null, and have the isActive method handle nulls.

Reply:
done